yarn.lock

```diff
@@ -2897,9 +2892,9 @@
     form-data "^3.0.0"
 
 "@types/node@*":
-  version "18.0.0"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-18.0.0.tgz#67c7b724e1bcdd7a8821ce0d5ee184d3b4dd525a"
-  integrity sha512-cHlGmko4gWLVI27cGJntjs/Sj8th9aYwplmZFwmmgYQQvL5NUsgVJG7OddLvNfLqYS31KFN0s3qlaD9qCaxACA==
+  version "17.0.23"
```

Comment:
I'm seeing some downgrades of packages in the yarn.lock file. Based on the current changes I don't think the yarn.lock files need to change, can we revert the changes to this file?